etl/steps/data/grapher/un/2024-10-21/census_dates.meta.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - State Capacity
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  years_since_last_census:
+    variables:
+      years_since_last_census:
+        title: Years since last census
+        description_short: The number of years since the last census of this country
+        unit: years
+        short_unit: "y"
```

Comment:
is this something we do for the year short units? it doesn't look super intuitive to me when I look at the chart especially since you are coding the color mapp based on whether it happened in the last 10 years

Reply:
I thought I had seen this before, but now that I checked it doesn't seem to be the case! I removed it :)